src/petals/utils/convert_block.py

```diff
@@ -16,13 +17,18 @@
 logger = get_logger(__name__)
 
 
+class QuantType(Enum):
+    NONE = 0
+    INT8 = 1  # 8-bit as in the LLM.int8() paper
+    NF4 = 2  # 4-bit as in the QLoRA paper
```

Comment:
Probably best to specify what specific nf4 config we're using, in case we ever need to switch to a different NF4

Reply:
This NF4 is NF4 with default bnb config, so it's a good candidate for just `NF4`. Indeed, we can do `NF4_V2` in future (`--quant_type` has a smart default and won't be manually chosen by a user too often, so it can have `_v2` in the name).